include/spdk/env.h

```diff
@@ -48,6 +48,7 @@ extern "C" {
 #include <stdio.h>
 
 struct spdk_pci_device;
+typedef int (*app_callback)(void * args);
```

Comment:
This needs an spdk_ prefix.

I also think it is simple enough that it can just be declared inline in the function declaration (so it doesn't need a name).